composer/trainer/trainer.py

```diff
@@ -1510,7 +1510,7 @@ def __init__(
 
         # Max Duration
         if max_duration is not None:
-            self.state.max_duration = ensure_time(max_duration, TimeUnit.EPOCH)
+            self.state.max_duration = ensure_time(max_duration, TimeUnit.EPOCH, allow_wct=False)
```

Comment:
why does this validation for WCT happen in helper fn but for scheduler it hapepns outside? we should be consistent